deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -389,6 +391,7 @@ spec:
 
     ## Operator versions
      - 4.1.0
+     - 4.0.0
```

Comment:
we have 4.0.1 also but not sure we need to mention it here or not?

Reply:
this is just for book keeping purpose, not really needed